web/meetup-facilitator/src/App.jsx

```diff
@@ -1,34 +1,58 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
 import './App.css'
+import Button from './components/Button'
+import { useRef, useState } from 'react'
 
 function App() {
-  const [count, setCount] = useState(0)
+
+  const buttonStyles = 'bg-green-700'
+  const groupName = useRef('')
+  const [shouldValidate, setShouldValidate] = useState(false);
+  const [isValid, setIsValid] = useState(true);
+
+
+  const updateGroupName = (e) => {
+    groupName.current = e.target.value;
+    console.log(groupName.current);
+  }
+
+  const onChange = (e) => {
+    updateGroupName(e);
+
+    if (shouldValidate) {
```

Comment:
I don't think there's a need for the `shouldValidate` state, wouldn't checking for `!isValid` have the same effect?